README.rst

```diff
@@ -13,11 +13,13 @@ A simple Result type for Python 3 `inspired by Rust
 <https://doc.rust-lang.org/std/result/>`__, fully type annotated.
 
 The idea is that a ``Result`` value can be either ``Ok(value)`` or ``Err(error)``,
```

Comment:
```suggestion
The idea is that a result value can be either ``Ok(value)`` or ``Err(error)``,
```

Reply:
Fixed in 6f70fbc0f3c323e6cc84a9e1aed6498246f89bd1